.github/CODEOWNERS

```diff
@@ -0,0 +1 @@
+* @confluentinc/c3-team
```

Comment:
it says there is some error, check other codeowners file

Reply:
Same line worked at https://github.com/confluentinc/blueway/blob/master/.github/CODEOWNERS
I am following up with devprod team at https://confluent.slack.com/archives/C09EP1SS3/p1715067008892029
will raise a PR/work seperately to fix this.
FYI this was present in the original PR by service bot as well https://github.com/confluentinc/control-center-images/pull/77/files

Reply:
**UPDATE:** Raised a PR to fix this at https://github.com/confluentinc/admin/pull/853

Reply:
**UPDATE:** This issue is resolved now.